src/Controller/CRUDController.php

```diff
@@ -1124,6 +1126,8 @@ protected function getRestMethod()
     }
 
     /**
+     * NEXT_MAJOR: move this method to 'sonata.admin.pool' service.
+     *
```

Comment:
IMO `configure()` method should be move to `sonata.admin.pool` with extra method like `getCurrentAdmin`. This will add independent between admin and `CRUDController` and allow split it in smaller `Action`.

Reply:
IMO making it mandatory to move this method to the pool is making both upgrade to sf 5 and sonata 4 harder to accomplish

Reply:
https://github.com/sonata-project/SonataAdminBundle/blob/d621ca4949b4db56b061d11125dfe29bd2d22ab8/src/Controller/CRUDController.php#L77-L82

`configure()` can be call in `getCurrentAdmin`. So technicly it can be done in one simply PR.